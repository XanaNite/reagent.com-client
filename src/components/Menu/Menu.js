import React from 'react'
import HamburgerMenu from '../HamburgerMenu/HamburgerMenu'

export default function Menu(){
    return (
        <div className='hamburger-menu'>
            <HamburgerMenu pageWrapId={"page-wrap"} outerContainerId={"App"} />
        </div>
    )
}
