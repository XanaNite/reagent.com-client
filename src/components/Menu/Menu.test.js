import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Menu from './Menu';

describe(`Menu component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders Menu by default', () => {
    const wrapper = shallow(<Menu />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the Menu given props', () => {
    const wrapper = shallow(<Menu {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
