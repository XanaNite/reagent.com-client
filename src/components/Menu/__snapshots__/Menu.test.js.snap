// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component renders Menu by default 1`] = `
<div
  className="hamburger-menu"
>
  <_default
    outerContainerId="App"
    pageWrapId="page-wrap"
  />
</div>
`;

exports[`Menu component renders the Menu given props 1`] = `
<div
  className="hamburger-menu"
>
  <_default
    outerContainerId="App"
    pageWrapId="page-wrap"
  />
</div>
`;
